aquadoggo/src/network/config.rs

```diff
@@ -13,6 +14,9 @@ use crate::network::identity::Identity;
 /// Key pair file name.
 const KEY_PAIR_FILE_NAME: &str = "private-key";
 
+/// The namespace used by the `identify` network behaviour.
+pub const NODE_NAMESPACE: &str = "aquadoggo";
```

Comment:
We might want to make this user-definable in the future. The namespace value is used when registering a rendezvous client with a rendezvous server and querying for other registered peers.

Reply:
Agreed, sharing a common identifier would be best. mDNS uses a 'service name' which is currently not user-configurable in the Rust implementation (see https://github.com/libp2p/rust-libp2p/issues/2898 for more detail).